scarlet/observation.py

```diff
@@ -22,54 +22,54 @@ def _centered(arr, newshape):
     return arr[tuple(myslice)]
 
 
-class Scene():
-    """Extent and characteristics of the modeled scence
+class Frame():
+    """Spatial and spectral characteristics of the data
+
     Attributes
     ----------
     shape: tuple
-        (bands, Ny, Nx) shape of the model image.
+        (bands, Ny, Nx) shape of the model image
     wcs: TBD
         World Coordinates
     psfs: array or tensor
         PSF in each band
-    filtercurve: TBD
-        Filter curve used for unpacking spectral information.
-    dtype: `~numpy.dtype`
-        Data type of the model.
+    bands: list of hashable elements
+        Names/identifiers of spectral bands
     """
-
-    #
-    def __init__(self, shape, wcs=None, psfs=None, filtercurve=None):
+    def __init__(self, shape, wcs=None, psfs=None, bands=None):
+        assert len(shape) == 3
         self._shape = tuple(shape)
         self.wcs = wcs
 
-        if psfs is not None:
-            psfs = np.array(psfs)
-            # Make sure that psf is always 3D
-            if len(psfs.shape) == 2:
-                psfs = psfs[None]
-            psfs = psfs / psfs.sum(axis=(1, 2))[:, None, None]
+        if psfs is None:
+            logger.warning('No PSFs specified. Possible, but dangerous!')
+        else:
+            assert len(psfs) == 1 or len(psfs) == shape[0]
```

Comment:
It's my crummy way to solve the problem of having one PSF for `Blend` and multiple ones for `Observation`. It would be nice to automate turning a 2D PSF image into a 3D, but you can't alter the `psfs` with a view (e.g. `psfs =  psfs.reshape(1,Ny,Nx)`) without breaking the comparison.

Reply:
Made more verbose